ghost/admin/app/styles/layouts/main.css

```
@@ -551,6 +551,10 @@ body:not(.gh-body-fullscreen) .gh-viewport {
     opacity: 1;
 }
 
+.gh-nav-list a:hover .gh-nav-member-count {
+    background: color-mod(var(--mainmenu-color-active-bg) l(-5%));
```

Biome (1.9.4)

[error] Unexpected unknown function: color-mod

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)